docs/tutorials/pachyderm-cat-dog.rst

```diff
@@ -130,9 +130,8 @@ You are now ready to create a project repo.
  Create Repos in Pachyderm for Training Data
 *********************************************
 
-To manage our training data effectively, we'll first need to create `repos
-<https://docs.pachyderm.com/latest/learn/basic-concepts/#basic-concepts-repositories-repo>`_ for
```

Comment:
This link also still works:  https://docs.pachyderm.com/products/mldm/latest/learn/basic-concepts/#basic-concepts-repositories-repo

Reply:
removed it due to its awkwardness